apps/src/applab/DesignToggleRow.jsx

```diff
@@ -92,7 +95,24 @@ module.exports = React.createClass({
       borderTopLeftRadius: 0
     }, (this.state.mode === Mode.CODE ? buttonPrimary : buttonSecondary));
 
-    if (this.state.mode === Mode.DESIGN) {
+    var showDataButtonStyle = $.extend({}, buttonStyle, buttonSecondary);
+
+    var iconStyle = {
+      margin: '0 0.3em'
+    };
+
+    if (this.state.mode === Mode.CODE) {
```

Comment:
FYI: This way of conditionally showing elements is okay, for small elements. In general, it is better to toggle it with style={display: none;} , so that React isn't swapping out huge numbers of elements into / out of the dom. (this is just my opinion, although it turns out to be necessary when the disappearing/reappearing child is a stateful react component).